test/performance/CMakeLists.txt

```diff
@@ -72,12 +71,13 @@ target_include_directories(
   tiledb_explore_msys_handle_leakage BEFORE PRIVATE
     ${TILEDB_CORE_INCLUDE_DIR}
     ${TILEDB_EXPORT_HEADER_DIR}
+    ${TILEDB_EP_INSTALL_PREFIX}/include
```

Comment:
This looks like it should also be an interface property of some library. What's not found if this line is absent? Is it cured when the test support library has its only include-directory property (see above)?

Reply:
This currently seems totally irrelevant, not sure of original intent,
removed